xtask/src/metrics.rs

```diff
@@ -82,11 +81,9 @@ impl Metrics {
     }
     fn measure_analysis_stats_path(&mut self, name: &str, path: &str) -> Result<()> {
         eprintln!("\nMeasuring analysis-stats/{}", name);
-        let output = Command::new("./target/release/rust-analyzer")
-            .args(&["analysis-stats", "--quiet", "--memory-usage", path])
-            .stderr(Stdio::inherit())
-            .output()?;
-        let output = String::from_utf8(output.stdout)?;
+        let output =
+            cmd!("./target/release/rust-analyzer analysis-stats --quiet --memory-usage {path}")
```

Comment:
So is this still needed?

Reply:
`cmd!` doesn't print the commands stderr to the console, so I temporarily replaced it with `std::process::Command`.

The actual problem was that the `eprintln!` in `print_memory_usage` was panicking, since it was detected as a debug print by the custom `eprintln` macro. Moving it to `cli.rs` fixes that.

Reply:
>doesn't print the commands stderr to the console, so I

fixed in latest version